src/views/EditSimple.vue

```diff
@@ -22,143 +23,145 @@
 
 <template>
 	<Popover ref="popover"
-		:open="isVisible"
+		:shown="isVisible"
 		:auto-hide="false"
 		:placement="placement"
-		:boundaries-element="boundaryElement"
-		open-class="event-popover"
-		trigger="manual">
-		<template v-if="isLoading">
-			<PopoverLoadingIndicator />
-		</template>
+		:boundary="boundaryElement"
+		popover-base-class="event-popover"
+		:triggers="[]">
+		<div class="event-popover__inner">
+			<template v-if="isLoading">
+				<PopoverLoadingIndicator />
+			</template>
 
-		<template v-else-if="isError">
-			<div class="event-popover__top-right-actions">
-				<Actions>
-					<ActionButton @click="cancel">
-						<template #icon>
-							<Close :size="20" decorative />
-						</template>
-						{{ $t('calendar', 'Close') }}
-					</ActionButton>
-				</Actions>
-			</div>
+			<template v-else-if="isError">
+				<div class="event-popover__top-right-actions">
+					<Actions>
+						<ActionButton @click="cancel">
+							<template #icon>
+								<Close :size="20" decorative />
+							</template>
+							{{ $t('calendar', 'Close') }}
+						</ActionButton>
+					</Actions>
+				</div>
 
-			<EmptyContent>
-				{{ $t('calendar', 'Event does not exist') }}
-				<template #icon>
-					<CalendarBlank :size="20" decorative />
-				</template>
-				<template #desc>
-					{{ error }}
-				</template>
-			</EmptyContent>
-		</template>
+				<EmptyContent>
```

Comment:
This needs to be adjusted to the new EmptyContent behaviour, as above. Has no `desc` or `default` slot anymore.